exporter/elasticsearchexporter/bulkindexer.go

```diff
@@ -428,8 +428,20 @@ func flushBulkIndexer(
 			ctx, int64(flushed), metric.WithAttributeSet(defaultAttrsSet),
 		)
 	}
+
+	fields := []zap.Field{}
```

Comment:
```suggestion
	var fields []zap.Field
```
nit: [prefer nil slice](https://www.jetbrains.com/help/inspectopedia/GoPreferNilSlice.html)